### DISCUSSION TITLE
Auth codes - organize by folder.

### DISCUSSION THREAD

**Discussion by edwardthomaswalker** [ente-io/ente#3044] [Enhancements]
I have dozens of MFA codes.  I would love the option to organize them like bookmarks into folders/subfolders/etc.
For example I could make a work folder and put the 1/2 dozen I use daily in there so I can get to them without having to search/scroll.

**Comment by nVitius** [comment]
Can you use tags for this? On the mobile app at least, you can filter your list by tag.

**Comment by ua741** [comment]
Closing this in favor of tag. We don't have any plans to introduce folder/subfolders within Ente Auth app.
